tensorflow/training/docker/1.15.4/py37/cu101/Dockerfile.gpu

```diff
@@ -0,0 +1,213 @@
+FROM nvidia/cuda:10.1-base-ubuntu18.04
```

Comment:
Why does 1.15.4 have cu101 while 1.15.3 has cu100?

Reply:
We updated the cuda version from 10.0 to 10.1 for 1.15.4